### PR TITLE
Add Rusty Playdate tooling/libs (WiP)

### PR DESCRIPTION
**Pull request by boozook**
Actually there are tooling and libraries, so 🤷🏻‍♂️

_Part of #1505_

### CODE REVIEW

**Comment by boozook** [comment]
@janhohenheim thank you! It would be super incredibly helpful. I just working on next release and trying to make it by the end of the month. And I also hoped to finish this PR after the release. But seriously not sure now 🤷🏻‍♂️.

File: content/news/051/index.md [hunk 1]
```diff
@@ -89,7 +109,7 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 <!-- Up to 10 most important notes + a link to the full details -->
 
-[See all meeting issues][label_meeting] including full text notes
+[See all meeting issues][label_meeting] including  full text notes
```
Comment (boozook) [inline content/news/051/index.md:112]:
```suggestion
[See all meeting issues][label_meeting] including full text notes
```

**Comment by boozook** [comment]
I've made the release but without one big feature that I'm still working on.
links:
- [post](https://gamedev.social/@playdaters/112498850850070371)
- [release](https://github.com/boozook/playdate/releases/tag/2024.05.24)

Currently working on:
- Assets-builder refactoring
- Support overridings for cargo-targets (bins, dev-targets such as examples)
- Fix issue when underlying cargo in the `cargo-playdate` touches host-cargo's cache and so triggering full-rebuild. Trying to rid of using underlying cargo's build-context.

**Comment by boozook** [comment]
And [here is what I mean](https://github.com/boozook/playdate/pull/366) as missed part of that release. That's just a first part😱

**Comment by janhohenheim** [comment]
@boozook what do you think about this? :)

**Comment by boozook** [comment]
@janhohenheim, Looks great, thank you so much! ☺️

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**